lib/models/user.go

```diff
@@ -11,6 +14,18 @@ type User struct {
 
 	Email          string `gorm:"not null;unique;size:128"`
 	HashedPassword string `gorm:"not null"`
+	Password       string `gorm:"-:all"`
```

Comment:
As the purpose of this field is to set the hashed password so[ ignore this password field on reading, writing and migrating ](https://gorm.io/docs/models.html#Field-Level-Permission)so that Gorm doesn't try to find the column named `password` on the database